src/App.js

```diff
@@ -1,9 +1,42 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
+import axios from "axios";
 import "./App.css";
+import Components from "./Components/Components.js"
 
 function App() {
+  const [NasaData,setNasaData] = useState([]);
+  const [date, setdate] = useState("");
+  const [explanation, setexplanation] = useState("");
+  const [mediatype, setmediatype] = useState();
+  const [title, settitle] = useState();
+  const [url, seturl] = useState();
+
+  useEffect(() => {
+    axios
+    .get ("https://api.nasa.gov/planetary/apod?api_key=QgbkPgyO0M5SCoeiV7M2Vp741LDB6R4QjYVxGbiT")
+
+  .then(Response =>
+    {settitle(Response.data.title);
+      setNasaData(Response.data.NasaData);
+      setdate(Response.data.date);
+      setexplanation(Response.data.explanation);
+      seturl(Response.data.url);
+      setmediatype(Response.data.mediatype);
+  })
```

Comment:
Instead of using all of these individual variables, we could have just had one `data` variable, and set that equal to `Response.data`. We also shouldn't capitalize `Response` because it is not a class or a component.